jp/components/overlay_position.md

```diff
@@ -1,160 +1,168 @@
-        this.overlay.onOpening.unsubscribe();
-        this.overlay.onClosed.unsubscribe();
-    }
-
-}
-```
+    | ターゲット          | horizontalDirection       |  verticalDirection       | horizontalStartPoint     | verticalStartPoint       |
+    |:----------------|:--------------------------|:-------------------------|:-------------------------|:-------------------------|
+    | new Point(0, 0) | HorizontalAlignment.Right | VerticalAlignment.Bottom | HorizontalAlignment.Left | VerticalAlignment.Bottom |
+<div class="divider"></div>
 
-### 水平または垂直開始点
-配置設定で水平開始点または垂直開始点を設定すると、コンテンツの開始位置を決定します。開始ポイントは [`positionSettings`]({environment:angularApiUrl}/interfaces/positionsettings.html) に渡された [`target`]({environment:angularApiUrl}/interfaces/positionsettings.html#target) が `HTMLElement` の場合のみ効果があり、[`AutoPositionStrategy`] ({environment:angularApiUrl}/classes/autopositionstrategy.html)、[`ElasticPositionStrategy`]({environment:angularApiUrl}/classes/elasticpositionstrategy.html)、[`ConnectedPositioningStrategy`] ({environment:angularApiUrl}/classes/connectedpositioningstrategy.html) でのみ使用できます。
-以下のデモで、オーバーレイ要素は選択した開始点に基づいてターゲット要素に配置します。方向は常に [`HorizontalAlignment.Right`] ({environment:angularApiUrl}/enums/horizontalalignment.html#right)、[`VerticalAlignment.Bottom`] ({environment:angularApiUrl}/enums/verticalalignment.html#bottom) です。
+4. **Auto** - **Connected** ポジション ストラテジと同じ方法で要素を配置します。要素が部分的にビューポートから出た場合を考慮して、異なる開始点も計算します。**Auto** ストラテジは最初に **Connected** ストラテジと同じように要素を表示しようとします。要素がビューポートから出た場合 **Auto** は開始点と方向を反転します。つまり、方向が 'bottom' の場合、要素は 'top' に切り替わります。反転後、要素がまだビューポートのものである場合、**Auto** は要素をビューポートにプッシュするために初期方向と開始点を使用します。たとえば、エレメントがビューポートの右側から 50 ピクセル分外に出た場合、**Auto** はそれを 50 ピクセル分左へプッシュします。その後、要素がビューポートから部分的に外れている場合、その高さまたは幅がビューポートより大きい場合、**Auto** は要素の左端/上端をビューポートの左端/上端に揃えます。デフォルト値:
+    | ターゲット          | horizontalDirection       |  verticalDirection       | horizontalStartPoint     | verticalStartPoint       |
+    |:----------------|:--------------------------|:-------------------------|:-------------------------|:-------------------------|
+    | new Point(0, 0) | HorizontalAlignment.Right | VerticalAlignment.Bottom | HorizontalAlignment.Left | VerticalAlignment.Bottom |
+<div class="divider"></div>
 
-<div class="sample-container loading" style="height: 400px">
-    <iframe id="overlay-position-sample-2-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/interactions/overlay-position-sample-2" onload="onSampleIframeContentLoaded(this);"></iframe>
-</div>
-<div>
-    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="overlay-position-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">Stackblitz で表示</button>
-</div>
+5. **Elastic** - **接続された**ポジション ストラテジのように要素を配置し、要素が部分的に見えなくなった場合にビューポートの内側に収まるように要素のサイズを変更します (幅や高さを再計算します)。[`minSize`]({environment:angularApiUrl}/interfaces/positionsettings.html#minsize) に渡すと、要素のサイズが特定のしきい値を下回る場合にサイズ変更を防ぐことができます。デフォルト値:
+
```

Comment:
Missing **positionSettings** ({environment:angularApiUrl}/interfaces/positionsettings.html) string:

In EN file - [`minSize`]({environment:angularApiUrl}/interfaces/positionsettings.html#minsize) can be passed in [`positionSettings`]({environment:angularApiUrl}/interfaces/positionsettings.html) to prevent resizing if it would put the element's dimensions below a certain threshold. 
In JP file - [`minSize`]({environment:angularApiUrl}/interfaces/positionsettings.html#minsize) に渡すと、要素のサイズが特定のしきい値を下回る場合にサイズ変更を防ぐことができます。